airmozilla/manage/views/users.py

```diff
@@ -39,12 +39,7 @@ def users(request):
             User.objects.filter(_mozilla_email_filter).count()
         ),
     }
-    form = forms.UserFindForm()
-    context = {
-        'form': form,
-        'users_stats': users_stats,
-    }
-    return render(request, 'manage/users.html', context)
```

Comment:
Keep the `context` variable. But just remove the `'form': form,` line. (and the `form = forms.UserFindForm()` line of course)